docs/development/git_chores.md

```diff
@@ -12,8 +12,7 @@ heads:
 ```
 ./build_tools/fetch_sources.py --remote --no-apply-patches
 # Capture new submodule heads, inspect and ensure things look sound.
-git add -A
-git commit -m "Rebase submodules (for conflict prep)"
+git commit -a -m "Rebase submodules (for conflict prep)"
```

Comment:
Do you also want to suggest using the new script here?

Reply:
I think not yet as this will need to advance a little before :)

Reply:
You could still advertise it but with a warning. Up to you.

Reply:
Done now :)